Docker/Dockerfile.content-watcher

```diff
@@ -37,4 +40,4 @@ RUN apt-get update && \
 
 EXPOSE 3000
 
-ENTRYPOINT [ "/usr/bin/tini", "--", "node", "dist/apps/api/main.js" ]
+ENTRYPOINT [ "/usr/bin/tini", "--", "node", "dist/apps/content-watcher/main.js" ]
```

Comment:
This the only one that doesn't use the `ENTRYPOINT [ "/usr/bin/tini", "--", "./docker-entrypoint.sh", "prod" ]`. Should it?

Reply:
I'd like to address Docker more in a subsequent PR. I think with this new monorepo setup, we could consolidate everything to a single Docker image instead of one per service, which would make our release/versioning process more manageable. But I didn't want to rock the boat too much in this PR, which is more about reorganizing than making substantive changes.